python/paddle/nn/layer/loss.py

```diff
@@ -1689,6 +1689,101 @@ def forward(self, input, positive, negative):
                                      name=self.name)
 
 
+class MultiMarginLoss(Layer):
+    r"""Creates a criterion that optimizes a multi-class classification hinge loss (margin-based loss) between input :math:`input` (a 2D mini-batch `Tensor`, in shape (N, C),
+        where C is number of classes) and label :math:`label` (which is a 1D tensor of target class indices,:math:`0 \leq label \leq \text{C}-1`):
+
+        For ith mini-batch sample, the loss in terms of the 1D input :math:`input_i` and scalar
+        output :math:`label_i` is:
+
+        .. math::
+            \text{loss}(input_i, label_i) = \frac{\sum^C_j \max(0, \text{margin} - input_i[label_i] + input_i[j])^p}{\text{C}}
+
+        where :math:`input_i \in \left\{0, \; \cdots , \; \text{C} - 1\right\}`
```

Comment:
I think here should be  math:`j \in \left\{0, \; \cdots , \; \text{C} - 1\right\}`, because j is the index to iterate over. `input_i` is a floating point vector.